drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/ScenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImpl.java

```diff
@@ -52,12 +52,20 @@ public void getItems(Consumer<List<KieAssetsDropdownItem>> assetListConsumer) {
         return response -> {
             List<KieAssetsDropdownItem> toAccept = response.stream()
                     .map(this::getKieAssetsDropdownItem)
+                    .filter(this::filterTargetDirectory)
```

Comment:
Could you please tell me why dont we use an Optional for for "toAccept"?
Isnt that possible that user would have some leftovers in target but removed all dmn files in the project?